### PR TITLE
Add `aiff` format

### PR DESCRIPTION
**Pull request by arthurflachs**

### CODE REVIEW

**Comment by sindresorhus** [comment]
See previous entries, like for example https://github.com/sindresorhus/file-type/commit/696957dc6dbc2541cb605054368f51f0fe310efb, for what's required.

File: index.js [hunk 1]
```diff
@@ -602,5 +602,12 @@ module.exports = input => {
 		};
 	}
 
+	if (check([0x46, 0x4F, 0x52, 0x4D, 0x00])) {
+		return {
+			ext: 'aif',
+			mime: 'audio/aif'
```
Comment (sindresorhus) [inline index.js]: Should be `audio/aiff`

File: index.js [hunk 2]
```diff
@@ -602,5 +602,12 @@ module.exports = input => {
 		};
 	}
 
+	if (check([0x46, 0x4F, 0x52, 0x4D, 0x00])) {
+		return {
+			ext: 'aif',
```
Comment (sindresorhus) [inline index.js]: Looks like `aiff` is the most common extension.

File: package.json [hunk 3]
```diff
@@ -101,7 +101,8 @@
 		"docx",
 		"pptx",
 		"xlsx",
-		"3gp"
+		"3gp",
+		"aif"
```
Comment (sindresorhus) [inline package.json]: aiff

File: readme.md [hunk 4]
```diff
@@ -148,6 +148,7 @@ It only needs the first 4100 bytes.
 - [`pptx`](https://en.wikipedia.org/wiki/Office_Open_XML)
 - [`xlsx`](https://en.wikipedia.org/wiki/Office_Open_XML)
 - [`3gp`](https://en.wikipedia.org/wiki/3GP_and_3G2)
+- [`aif`](https://en.wikipedia.org/wiki/Audio_Interchange_File_Format)
```
Comment (sindresorhus) [inline readme.md]: aiff

File: test.js [hunk 5]
```diff
@@ -75,7 +75,8 @@ const types = [
 	'docx',
 	'pptx',
 	'xlsx',
-	'3gp'
+	'3gp',
+	'aif'
```
Comment (sindresorhus) [inline test.js]: aiff

File: index.js (hunk 2 above)
Comment (sindresorhus) [reply index.js]: Ok

**Comment by sindresorhus** [comment]
@mifi The inconsistency is for a good reason. The official name is AIFF, but common extension used is `.aif`. It's like JPEG and `.jpg`.

**Comment by sindresorhus** [comment]
@arthurflachs Can you fix the merge conflict?

**Comment by mifi** [comment]
@sindresorhus Oh, I thought that test.js, package.json and readme.md should contain the extension, like we use ‘jpg’ for jpeg.

**Comment by sindresorhus** [comment]
@mifi Good point. Yes, it should for consistency. I've fixed that now.

**Comment by mifi** [comment]
Cool, then this looks good to me👍

**Pull request closed without merging**